README.md

```diff
@@ -1,4 +1,7 @@
-Kooper [![Build Status](https://travis-ci.org/spotahome/kooper.svg?branch=master)](https://travis-ci.org/spotahome/kooper)
+Kooper
+[![Build Status](https://travis-ci.org/spotahome/kooper.svg?branch=master)](https://travis-ci.org/spotahome/kooper)
+[![Go Report Card](https://goreportcard.com/badge/github.com/spotahome/kooper)](https://goreportcard.com/report/github.com/spotahome/kooper)
+[![GolangCI](https://golangci.com/badges/github.com/spotahome/kooper.svg)](https://golangci.com)
```

Comment:
Thanks for the PR!

GolangCI badge and Go Report card have almost the same information (Gofmt, Goimports, Golint...). I would stick with only one. In my case I prefer Go report card. Everything else is perfect!